otter/integration/lib/trial_tools.py

```diff
@@ -98,6 +107,66 @@ def get_resource_mapping():
     return res
 
 
+def filter_logs(observer):
+    """
+    Filter out logs like
+    "Starting factory <twisted.web.client_HTTP11ClientFactory".
+    """
+    def filter(eventdict):
```

Comment:
Non-blocking nitpick: Could you rename this to `observer` or `filter_starting_messages` since `filter` is a predefined function?